README.md

```diff
@@ -8,15 +34,33 @@ error_log /var/log/nginx/error.log debug;
 ```
 
 Démarrer nginx
-``` shell
-erb nginx.conf.erb > nginx.conf
-docker run -v $(pwd)/nginxbase.conf:/etc/nginx/nginx.conf:ro -v $(pwd)/nginx.conf:/etc/nginx/conf.d/default.conf:ro -p 80:80 --entrypoint nginx-debug nginx '-g daemon off;' 2>&1 |egrep '^(Host: |X-Forwarded-Host: |.GET .* HTTP)'
+```shell
+  # poser les variables d'env utilisées dans nginx.conf.erb
+  export FRONT_REVIEW_APP_NAME_PREFIX=pix
+  export API_HOST_SUFFIX=osc-fr1.scalingo.io
+  #
+  erb nginx.conf.erb > nginx.conf
+  docker rm review-router
```

Comment:
Proposition : 

Faire un docker-compose simple qui lance un test de configuration nginx, et rendre le nginx debug optionnel :

```
services:
    init-nginx:
        image: nginx
        entrypoint: 'nginx'
        command: '-t'
        volumes:
          - ./nginxbase.conf:/etc/nginx/nginx.conf:ro
          - ./nginx.conf:/etc/nginx/conf.d/default.conf:ro
    review-router:
        image: nginx
        entrypoint: "nginx${DEBUG-}"
        command: '-g "daemon off;"'
        volumes:
          - ./nginxbase.conf:/etc/nginx/nginx.conf:ro
          - ./nginx.conf:/etc/nginx/conf.d/default.conf:ro
        ports:
          - 80:80
        depends_on:
          init-nginx:
            condition: service_completed_successfully
```

Fonctionnement classique : `docker-compose up`
Fonctionnement debug : `DEBUG='-debug' docker-compose up`

Reply:
I love this